### ISSUE TITLE
KEGG has introduced new terms

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#456]
### Bug descriptions

Previous KEGG issues:  #99 #316 

the kegg module is on hold because of issues related to the structure of hte kegg obo:
https://github.com/tripal/tripal_analysis_kegg/issues/18

We said hey thats fine, we'll just use our pre-existing loaded KEGG ontology.  However, the latest round of annotations have new kegg accessions, so we can no longer load unless we a) add the term ourselve s(dangerous and obnoxious because we'll want the relationship info too) or b) wait for the tripal analysis KEGG issue to get fixed.

**Comment by bradfordcondon** [comment]
issue where the new term is mentioned

https://github.com/statonlab/hardwoods_site/issues/340#issuecomment-443843240

**Comment by almasaeed2010** [comment]
Does KEGG provide versions every time they update? If so, can we load each version in? Then run kegg on the new organisms only without losing any of the pre-annotated organisms?

**Comment by bradfordcondon** [comment]
>Does KEGG provide versions every time they update? 

i dont think so, no.  I believe it will always be a "discover because it broke" situation.  Adding new terms would normally be no problem, except that the KEGG OBO breaks the newer tripal OBO loader and stephen wants a deeper fix.

Re-loading hte kegg ontology wont impact pre-annotated organisms.

**Comment by jwest60** [comment]
https://www.genome.jp/kegg/docs/upd_map.html

This is a history of updates for the pathway module, it looks like new terms are getting added once or twice a month

**Comment by bradfordcondon** [comment]
awesome find!  is it just for pathway though or can we find one for all the subontologies...

**Comment by jwest60** [comment]
it looks like you can just browse through the tabs at the top of [this page](https://www.genome.jp/kegg/brite.html) and click the update history link on each module
